tests/unit/Types/ContextFactoryTest.php

```diff
@@ -57,7 +57,10 @@ public function testReadsAliasesFromClassReflection()
             ];
             $context = $fixture->createFromReflector(new ReflectionClass($this));
 
-            $this->assertSame($expected, $context->getNamespaceAliases());
+            $actual = $context->getNamespaceAliases();
+
+            // sort so that order differences don't break it
+            $this->assertSame(sort($expected), sort($actual));
```

Comment:
The reordering of the `use` imports explains why I had to add the `sort()` behavior around the two arrays here... I thought that easier than trying to _assume_ the ordering of the array members at runtime.  It now looks like the runtime ordering actually was consistent and based on the import ordering.